python.json

```diff
@@ -70,5 +70,29 @@
   {"id": "py.django-suit", "imports": ["suit"], "name": "Django Suit", "repo": "darklow/django-suit", "tags": ["django-admin"], "tech": ["app-blocks"]},
   {"id": "py.django-xadmin", "imports": ["xadmin"], "name": "Xadmin", "repo": "sshwsfc/xadmin", "tags": ["django-admin"], "tech": ["app-blocks"]},
   {"id": "py.flask-admin", "imports": ["flask_admin"], "name": "Flask-Admin", "repo": "flask-admin/flask-admin", "tags": ["flask-admin"], "tech": ["app-blocks"]},
-  {"id": "py.django-grappelli", "imports": ["grappelli"], "name": "Django Grappelli", "repo": "sehmaschine/django-grappelli", "tags": ["django-admin"], "tech": ["app-blocks"]}
+  {"id": "py.django-grappelli", "imports": ["grappelli"], "name": "Django Grappelli", "repo": "sehmaschine/django-grappelli", "tags": ["django-admin"], "tech": ["app-blocks"]},
+
+  {"id": "py.pypattyrn", "imports": ["pypattyrn"], "name": "PyPattyrn", "repo": "tylerlaberge/PyPattyrn", "tags": ["patterns"], "tech": ["app-arch"]},
+  {"id": "py.python-patterns", "imports": ["behavioral", "creational", "dft", "fundamental", "other", "structural"], "name": "python-patterns", "repo": "faif/python-patterns", "tags": ["patterns"], "tech": ["app-arch"]},
+  {"id": "py.sorted-containers", "imports": ["sortedcontainers"], "name": "Sorted Containers", "repo": "grantjenks/python-sortedcontainers", "tags": ["containers"], "tech": ["data-structures"]},
```

Comment:
Well, they also refer to it as 'sorted containers' in the text.

Usually, I try to pick a shorter name/tag etc, because I assume that real estate on screen to display those will be limited. It's better to see "Sorted Containers" than "Python Sorted..." as a label.

Reply:
ok, fair enough